IllegalArgumentException in SecureXmlUtil

When Apache Xalan is on the class path an uncaught `IllegalArgumentException` is thrown by `SecureXmlUtil`:
```
java.lang.ExceptionInInitializerError
                at org.eclipse.milo.opcua.stack.core.serialization.OpcUaXmlStreamDecoder.<init>(OpcUaXmlStreamDecoder.java:86)
                at org.eclipse.milo.opcua.sdk.server.namespaces.loader.VariableNodeLoader.loadNode0(VariableNodeLoader.java:71)
                at org.eclipse.milo.opcua.sdk.server.namespaces.loader.VariableNodeLoader.loadAllNodes(VariableNodeLoader.java:12987)
                at org.eclipse.milo.opcua.sdk.server.namespaces.loader.NodeLoader.loadNodes(NodeLoader.java:33)
                at org.eclipse.milo.opcua.sdk.server.namespaces.OpcUaNamespace.loadNodes(OpcUaNamespace.java:152)
                at org.eclipse.milo.opcua.sdk.server.namespaces.OpcUaNamespace.lambda$new$2(OpcUaNamespace.java:88)
                at org.eclipse.milo.opcua.sdk.server.LifecycleManager$1.startup(LifecycleManager.java:95)
                at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807)
                at org.eclipse.milo.opcua.sdk.server.LifecycleManager.onStartup(LifecycleManager.java:49)
                at org.eclipse.milo.opcua.sdk.server.AbstractLifecycle.startup(AbstractLifecycle.java:32)
                at org.eclipse.milo.opcua.sdk.server.api.ManagedNamespaceWithLifecycle.startup(ManagedNamespaceWithLifecycle.java:42)
                at org.eclipse.milo.opcua.sdk.server.OpcUaServer.<init>(OpcUaServer.java:122)
   
Caused by: java.lang.IllegalArgumentException: Not supported: http://javax.xml.xmlconstants/property/accessExternalDTD
                at org.apache.xalan.processor.TransformerFactoryImpl.setAttribute(TransformerFactoryImpl.java:571)
                at org.eclipse.milo.opcua.stack.core.util.SecureXmlUtil.<clinit>(SecureXmlUtil.java:49)
```

This should be caught and logged like in `SecureXmlUtil.trySetFeatures`.

--- a/opc-ua-stack/stack-core/src/main/java/org/eclipse/milo/opcua/stack/core/util/SecureXmlUtil.java
+++ b/opc-ua-stack/stack-core/src/main/java/org/eclipse/milo/opcua/stack/core/util/SecureXmlUtil.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 the Eclipse Milo Authors
+ * Copyright (c) 2022 the Eclipse Milo Authors
  *
  * This program and the accompanying materials are made
  * available under the terms of the Eclipse Public License 2.0
@@ -46,8 +46,8 @@ public class SecureXmlUtil {
         trySetFeature("http://xml.org/sax/features/external-parameter-entities", false);
         trySetFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
 
-        SHARED_TRANSFORMER_FACTORY.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
-        SHARED_TRANSFORMER_FACTORY.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");
+        trySetAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+        trySetAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");
     }
 
     private static void trySetFeature(String feature, boolean value) {
@@ -55,7 +55,16 @@ public class SecureXmlUtil {
             SHARED_DOCUMENT_BUILDER_FACTORY.setFeature(feature, value);
         } catch (Exception e) {
             LoggerFactory.getLogger(SecureXmlUtil.class)
-                .debug("Error configuring feature: " + feature, e);
+                .warn("Error configuring feature: " + feature + "=" + value, e);
+        }
+    }
+
+    private static void trySetAttribute(String attribute, Object value) {
+        try {
+            SHARED_TRANSFORMER_FACTORY.setAttribute(attribute, value);
+        } catch (Exception e) {
+            LoggerFactory.getLogger(SecureXmlUtil.class)
+                .warn("Error configuring attribute: " + attribute + "=" + value, e);
         }
     }
 

